### ISSUE TITLE
Info on LR

### ISSUE THREAD

**Issue by ankur7721** [hpanwar08/detectron2#21]
Hi @hpanwar08 

Can you please confirm, what is the LR (learning rate you used), also did you used same learning rate through out your training process or was it scheduled at different learning rates ?

**Comment by hpanwar08** [comment]
Hi,
Learning rate are mentioned in the config files and there is multistep lr. I trained on constant learning rate except of startup.

https://github.com/hpanwar08/detectron2/blob/b86e2e76288e50df09cd389245c233ad8998635c/configs/DLA_mask_rcnn_R_50_FPN_3x.yaml#L16

https://github.com/hpanwar08/detectron2/blob/b86e2e76288e50df09cd389245c233ad8998635c/detectron2/config/defaults.py#L486

**Comment by ankur7721** [comment]
thanks @hpanwar08 
Since you have trained model on train-0.zip, train-1.zip, train-2.zip, train-3.zip
If I extend it for train-4.zip, train-5.zip, train-6.zip...
Do you recommend any changes in

1. Num_of_steps
2. LR

**Comment by ankur7721** [comment]
thanks @hpanwar08 for the LR info

Since you have trained model on train-0.zip, train-1.zip, train-2.zip, train-3.zip
If I extend it for train-4.zip, train-5.zip, train-6.zip...
Do you recommend any changes in

1. Num_of_steps
2. LR

**Comment by hpanwar08** [comment]
MAX_ITER would depend on how long you can train the model. If there is not limitation on the training time then train for atleast 3 epochs (3 * num of images)
If your batch size is 1 then keep your learning rate low, maybe start with 0.0001 and check the convergence.

Create a new config yaml file for training.

**Comment by ankur7721** [comment]
thanks @hpanwar08

**Comment by ankur7721** [comment]
hi @hpanwar08 

how do I trim the train.json to only include annotations from train-4, train-5 and train-6 ?

**Comment by ankur7721** [comment]
hi @hpanwar08 

how do I trim the train.json to only include annotations from train-4, train-5 and train-6 ?

**Comment by hpanwar08** [comment]
train.json contains all the annotations of PublayNet. You have to iterate through the images and json to extract the relevant annotations.
I have trained for approx 1.5 epochs

**Comment by ankur7721** [comment]
thanks
